### PR TITLE
Home_task 6 for review

### PR DESCRIPTION
**Pull request by NatalliaSem**

### CODE REVIEW

File: Home_task_6.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                        os.chdir(original_path)
+                        is_true = False
+                        break
+                    except FileNotFoundError:
+                        print('No file with such name. Try again')
+                    except PermissionError:
+                        print('No file with such name. Try again')
+            else:
+                print('Try again here')
+        num_records = int(input("Enter records num:\n"))
```
Comment (Verantina) [inline Home_task_6.py:132]: What will happen if the number is more then number of records in the file?

File: Home_task_6.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    def __init__(self, city, weath_message):
+        self.weath_message=weath_message
+        self.city = city
+
+    def weather_pred(self):
+        temperature = random.randint(1, 31)
+        message1 = f'Weather-----------------------\nWeather forecast for: {self.city}\n' \
+                  f'{self.weath_message}\n' \
+                  f'temperature +{temperature} Celsius degrees \n' \
+                  f'----------------------------------\n\n'
```
Comment (Verantina) [inline Home_task_6.py:73]: Please fix it, just have somewhere separate class for creation of this string. The same comment as for the previous HomeWork

File: Home_task_6.py (hunk 1 above)
Comment (NatalliaSem) [reply Home_task_6.py]: fixed

File: Home_task_6.py (hunk 2 above)
Comment (NatalliaSem) [reply Home_task_6.py]: fixed

File: Home_task_6_new.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
+from datetime import datetime, timedelta, date
```
Comment (Verantina) [inline Home_task_6_new.py:1]: Can't see any changes, only normalize function added

**Review by Verantina — APPROVED** [review]

**Pull request merged**